src/Symfony/Bridge/Monolog/Handler/ServerLogHandler.php

```diff
@@ -48,7 +48,7 @@ protected function getDefaultFormatter()
 /**
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  *
- * @internal since Symfony 6.1
+ * @internal
  */
 trait ServerLogHandlerTrait
```

Comment:
@lyrixx @Seldaek WDYT of removing support for monolog < 3?

Reply:
Yes that would allow some conditional code mess to be cleaned up which sounds great

Reply:
@lyrixx up for a PR doing so?